src/Raven.Server/Utils/RavenCli.cs

```diff
@@ -67,90 +69,225 @@ internal class RavenCli
                 writer.Write(msg);
         };
 
-        private List<string> _promptArgs = new List<string> { "ravendb" };
+
         private TextWriter _writer;
         private TextReader _reader;
+        private RavenServer _server;
+        private bool _experimental;
+        private bool _consoleColoring;
+
+        private enum Command
+        {
+            // ReSharper disable once UnusedMember.Local
+            None,
+            Prompt,
+            HelpPrompt,
+            Quit,
+            Log,
+            Clear,
+            ResetServer,
+            Stats,
+            Info,
+            Gc,
+            LowMem,
+            Help,
+            Logo,
+            Experimental,
+            ImportDir,
+            CreateDb,
+            Logout,
```

Comment:
logout - close gracefully remote cli connection

importdir + createdb : are bypassing REST so they aren't listed in help and cannot be used unless "Experimental on" command is explicitly entered.